admin-dev/themes/default/js/bundle/default.js

```diff
@@ -4,6 +4,19 @@
 $(document).ready(function() {
     $('[data-toggle="tooltip"]').tooltip();
     rightSidebar.init();
+    /** spinner loading */
+    $(this).ajaxStart(function ()
```

Comment:
Consider storing `$(this)` into a variable, in order to avoid repeating the method call three times.
Same for `$('#ajax_spinner')`:

```js
var $this = $(this);
var $ajaxSpinner = $('ajax_spinner');

$this.ajaxStart(function () {
  $ajaxSpinner.show();
});
$this.ajaxStop(function () {
  $ajaxSpinner.hide();
});
// ...
```

Also, I think that opening braces should not be on a line of their own (at least for consistency's sake).